### PR TITLE
Add AdafruitIO with Feather M0 Guide

### PR DESCRIPTION
**Pull request by kent-williams**

### CODE REVIEW

File: docs/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruitio.mdx [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+Before we begin, please make sure you've followed the steps from
+the Console quickstart [here](/use-the-network/console/quickstart).
+This will ensure you have a Console account and have created a new
+device.
+
+:::
+
+## Objective and Requirements
+
+In this guide, you will learn:
```
Comment (PharkMillups) [inline docs/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruitio.mdx]:
```suggestion
In this guide, you will learn how to:
```

File: docs/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruitio.mdx [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+
+* How to connect your hardware
```
Comment (PharkMillups) [inline docs/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruitio.mdx]:
```suggestion
* Connect your hardware
```

File: docs/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruitio.mdx [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+* Setup your firmware environment
+* How to program a basic application that will send packets over the Helium Network
```
Comment (PharkMillups) [inline docs/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruitio.mdx]:
```suggestion
* Program a basic application that will send packets over the Helium Network
```

File: docs/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruitio.mdx [hunk 4]
```diff
@@ -0,0 +1,345 @@
...
+  )}
+/>
+
+### Adding the Antenna
+
+Please visit Adafruit's Documentation for antenna options and instructions [here](https://learn.adafruit.com/adafruit-feather-m0-radio-with-lora-radio-module/antenna-options).
+
+### Jumping IO1 to Pin6
+
+In order to use the RFM95 radio module onboard the Feather M0 for LoRaWAN we'll need to connect two pins together.  The pins are labeled io1 and 6 on the board, as shown below. If you have the headers soldered on, you can use a female to female jumper cable to connect them, or you can simply solder a wire from pin to pin.
```
Comment (PharkMillups) [inline docs/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruitio.mdx]:
```suggestion
In order to use the RFM95 radio module onboard the Feather M0 for LoRaWAN we'll need to connect two pins together.  The pins are labeled `io1` and `6` on the board, as shown below. If you have the headers soldered on, you can use a female to female jumper cable to connect them, or you can simply solder a wire from pin to pin.
```

File: docs/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruitio.mdx [hunk 5]
```diff
@@ -0,0 +1,345 @@
...
+
+<img
+  src={useBaseUrl(
+    "img/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruit-feather-m0-rfm95-jumper-pins.jpg"
+  )}
+/>
+
+### Wiring the DHT22 Sensor
+
+Connect the signal pin 2 of the DHT22 sensor to pin 12 on the feather board. You will also need to
```
Comment (PharkMillups) [inline docs/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruitio.mdx]:
```suggestion
Connect the signal pin `2` of the DHT22 sensor to pin `12` on the feather board. You will also need to 
```

File: docs/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruitio.mdx [hunk 6]
```diff
@@ -0,0 +1,345 @@
...
+<img
+  src={useBaseUrl(
+    "img/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruit-feather-m0-rfm95-jumper-pins.jpg"
+  )}
+/>
+
+### Wiring the DHT22 Sensor
+
+Connect the signal pin 2 of the DHT22 sensor to pin 12 on the feather board. You will also need to 
+add a pull up resister (20-50k) between the signal pin 2 of the DHT22 sensor and the 3.3V supply.
```
Comment (PharkMillups) [inline docs/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruitio.mdx]:
```suggestion
add a pull up resister (20-50k) between the signal pin `2` of the DHT22 sensor and the 3.3V supply.
```

File: docs/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruitio.mdx [hunk 7]
```diff
@@ -0,0 +1,345 @@
...
+
+Next we'll need to fill in the AppEUI\(lsb\), DevEUI\(lsb\), and AppKey\(msb\), in the sketch, which you can find on the device details page on Console. Be sure to use the formatting buttons to match the endianess and formatting required for the sketch, shown below.
+
+<img
+  src={useBaseUrl(
+    "img/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruit-feather-m0-rfm95-console-keys.png"
+  )}
+/>
+
+At the top of the sketch, replace the three **FILL\_ME\_IN** fields, with the matching field from Console, example shown below.
```
Comment (PharkMillups) [inline docs/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruitio.mdx]: Would it be better to have an embedded code sample here people can copy and paste?

**Review by PharkMillups — DISMISSED** [review]
Left a few comments and suggestions.. Looks great otherwise.

File: docs/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruitio.mdx (hunk 7 above)
Comment (kent-williams) [reply docs/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruitio.mdx]: It's much too long unfortunately and we would need to update the docs if any changes are made to the code. Perhaps once we find a good way of pulling it from github and collapsing the view.

Comment (PharkMillups) [reply docs/use-the-network/devices/development/adafruit-feather-m0-rfm95/adafruitio.mdx]: Roger. Screenshot works.

**Review by PharkMillups — APPROVED** [review]

**Pull request merged**